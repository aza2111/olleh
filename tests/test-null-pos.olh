//testing null comparison

list<char> lis;

if (lis == Null):
	if (Null == lis):
		print "it's null";
	end;
end;

list<list> lis2;
if (lis2 == Null):
	if (Null == lis2):
		print "it's null";
	end;
end;


map<char> xx;
if (xx == Null):
	if (Null == xx):
		print "it's null";
	end;
end;

map<String> xxx;
if (xxx == Null):
	if (Null == xxx):
		print "it's null";
	end;
end;


String x;
if (x == Null):
	if (Null == x):
		print "it's null";
	end;
end;

