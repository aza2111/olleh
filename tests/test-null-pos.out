it's null
it's null
it's null
it's null
it's null
